includes/Jobs/RequestWikiRemoteAIJob.php

```diff
@@ -249,10 +274,25 @@ private function handleLiveRun( string $outcome, string $comment ): bool {
 	private function queryOpenAI(
 		string $sitename,
 		string $subdomain,
-		string $reason
+		string $reason,
+		string $username,
+		string $language,
+		bool $bio,
+		bool $private,
+		string $category,
+		bool $nsfw,
+		string $nsfwtext
 	): ?array {
 		try {
-			$sanitizedReason = "Wiki name: $sitename. Subdomain: $subdomain. Wiki request reason: " .
+			$isBio = $bio ? "Yes" : "No";
+			$isPrivate = $private ? "Yes" : "No";
+			$isNsfw = $nsfw ? "Yes" . $nsfwtext : "No";
+			$nsfwReasonText = $nsfw ? "What type of NSFW content will it feature? '$nsfwtext'. " : "";
+
+			$sanitizedReason = "Wiki name: '$sitename'. Subdomain: '$subdomain'. Requester: '$username'. " .
+			"Language: '$language'. Focuses on real people/groups? '$isBio'. Private wiki? '$isPrivate'. " .
+			"Category: '$category'. Contains content that is not safe for work? '$isNsfw'. " .
+			$nsfwReasonText . "Wiki request reason: " .
```

Comment:
Oh though this seeks like a real problem that will significantly confuse the AI?